ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/theme/ExchangeRateColors.kt

```diff
@@ -0,0 +1,34 @@
+@file:Suppress("MagicNumber")
+
+package co.electriccoin.zcash.ui.design.theme
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+
+@Immutable
+data class ExchangeRateColors(
```

Comment:
As discussed internally, we should split this class make it part of the new alternative `ZashiColors` and make it internal